client/app/scripts/utils/web-api-utils.js

```diff
@@ -235,20 +241,19 @@ export function getResourceViewNodesSnapshot(state, dispatch) {
   getNodesForTopologies(state, dispatch, topologyIds);
 }
 
-// NOTE: getState is called every time to make sure the up-to-date state is used.
-export function getTopologies(getState, dispatch, initialPoll = false) {
+function pollTopologies(getState, dispatch, initialPoll = false) {
```

Comment:
Would it be tricky to write `pollTopologies` in terms of `getTopologiesOnce`?

I'm trying to get over my DRYOCD so maybe this is not important. `reqwest` seems to support Promises but we're not using them very much.